### PR TITLE
MC-34170 Add Note about PHP 7.4.2 bug

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) adds a warning about a rendering bug in PHP 7.4.2.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html

### CODE REVIEW

File: src/guides/v2.4/install-gde/system-requirements-tech.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ Magento only uses MySQL features compatible with MariaDB. MariaDB may not be com
 
 While Magento supports PHP 7.3, Magento 2.4 is tested on, and we recommend using, PHP 7.4.
 
+{.bs-callout-warning}
+There is a [bug](https://bugs.php.net/bug.php?id=79174) in PHP 7.4.2 which causes some messages to not render properly. Magento recommends using other versions of 7.4.
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/system-requirements-tech.md:51]: Should we just say that Magento does not support 7.4.2?

**Comment by dshevtsov** [comment]
running tests

**Comment by dshevtsov** [comment]
This test run is intended for CI/CD testing.

File: src/guides/v2.4/install-gde/system-requirements-tech.md (hunk 1 above)
Comment (dobooth) [reply src/guides/v2.4/install-gde/system-requirements-tech.md:51]: I am reticent to put that right below the line that says we support 7.4. Pointing them to the bug shows it's not us and it's been fixed.

**Comment by dobooth** [comment]
running tests

**Pull request merged**